### ISSUE TITLE
Update uuid library.

### ISSUE THREAD

**Issue by danilocubos** [Baseflow/flutter_cached_network_image#562]
## 🏗 Enhancement Proposal

Update `uuid` (https://pub.dev/packages/uuid) library to the latest version. Today it is 3.0.1.

### Reason

Trying to run `flutter pub get` with both `cached_network_image: ^2.5.0` and `uuid ^3.0.1` with `flutter 2.0.0` throws the following error:

```
pub get failed (1; So, because my_project depends on both cached_network_image ^2.5.0 and uuid ^3.0.1, version solving failed.)
```
### Platforms affected 
- [ X ] :iphone: iOS
- [ X ] :robot: Android
- [ X ] Any other

**Comment by cto-leaps** [comment]
+1 please 🙏

**Comment by renefloor** [comment]
Should be fixed in cached_network_image: ^2.5.1
